builder/qemu/builder_test.go

```diff
@@ -83,7 +84,7 @@ func TestBuilderPrepare_Defaults(t *testing.T) {
 		t.Errorf("bad ssh port: %d", b.config.Comm.SSHPort)
 	}
 
-	if b.config.VMName != "packer-foo" {
```

Comment:
I will need to document this as a BC change. The BC change should only affect **default** filenames since 0.8.0, reverting to behavior that more closely reflects the naming pattern from 0.7.5. This was an undocumented change between 0.7.5 and 0.8.0. **Custom** filenames from 0.8.0 forward are not changed.